internal/cmd/iam/command_pool_create.go

```diff
@@ -18,19 +17,18 @@ func (c *identityPoolCommand) newCreateCommand() *cobra.Command {
 		RunE:  c.create,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Create an identity pool named "DemoIdentityPool" with provider "op-12345":`,
-				Code: `confluent iam pool create DemoIdentityPool --provider op-12345 --description new-description --identity-claim claims.sub --filter 'claims.iss=="https://my.issuer.com"'`,
+				Text: `Create an identity pool named "demo-identity-pool" with provider "op-12345":`,
```

Comment:
What kind of a provider is this? Wondering if there's more specific terminology we can use
```suggestion
				Text: `Create an identity pool named "demo-identity-pool" with identity provider "op-12345":`,
```

Reply:
According to: https://docs.confluent.io/cloud/current/api.html#tag/Identity-Providers-(iamv2) they are "OAuth/OpenID Connect" identity providers.